pkg/providers/qemu/stop_instance_cgo.go

```diff
@@ -23,7 +24,12 @@ func (p *QemuProvider) StopInstance(id string) error {
 		return errors.New("invalid instance id (should be qemu pid)", err)
 	}
 
-	if err := syscall.Kill(pid, syscall.SIGKILL); err != nil {
+	process, err := os.FindProcess(pid)
+	if err != nil {
+		return errors.New("failed to find process", err)
```

Comment:
we don't necessarily want to return here; we just want to throw a warning and move on (for example if the process was stopped externally). we still want to reach the final line `return p.state.RemoveInstance(instance)`

Reply:
I have updated the PR. Please review.